Makefile

```diff
@@ -0,0 +1,9 @@
+install: install-glide
+	( cd maptime ; glide install )
+
+install-glide:
```

Comment:
Could you use https://github.com/m3db/ci-scripts as a submodule, and re-use the targets in `common.mk` for glide. E.g. from [m3cluster](https://github.com/m3db/m3cluster/blob/master/Makefile#L2)

Reply:
Once you land this PR, could you add a target to install this linter in `common.mk` (so other m3db repos can re-use that instead of copy-pasting the same snippet). 

Not sure how best to handle the circular dependency. Shouldn't be an issue because this repo doesn't need to install itself but maybe something to think about.